mod/format/set/all/html/views.rb

```diff
@@ -3,23 +3,12 @@
 format :html do
   def prepare_content_slot
     class_up "card-slot", "d0-card-content"
-    voo.hide :menu
+    voo.hide :menu, :edit_button
```

Comment:
This is definitely an improvement.

Arguably, the more appropriate API for this is `optional: :hide`, eg:
```
wrap do
  [
    render_menu(optional: :hide), 
    render_core, 
    render_edit_button(optional: :hide, edit: :inline)
  ]
end
```

Also, not necessarily for this release, but it kind of seems like we could get rid of `content_with_title`, too, using a show/hide approach.

Reply:
That changes the view cache key. I'd rather don't mess with the view cache.